bigstore/bigstore.py

```diff
@@ -322,9 +324,18 @@ def pull():
 
                         break
 
-    sys.stderr.write("pushing bigstore metadata...")
-    g().push("origin", "refs/notes/bigstore")
-    sys.stderr.write("done\n")
+    if g().diff('refs/notes/bigstore', 'refs/notes/bigstore-remote'):
```

Comment:
Sure. If you look at 279 and 282 above, some updates are made to the notes ref (which tracks file downloads, etc.). We then push to update the notes ref of the remote repo to keep it up to date.